lib/manageiq/environment.rb

```diff
@@ -9,16 +9,16 @@ def self.manageiq_plugin_setup(plugin_root = nil)
       # determine plugin root dir. Assume we are called from a 'bin/' script in the plugin root
       plugin_root ||= Pathname.new(caller_locations.last.absolute_path).dirname.parent
 
-      manageiq_plugin_update(plugin_root)
+      manageiq_plugin_update(plugin_root, force: false)
```

Comment:
```suggestion
      manageiq_plugin_update(plugin_root, force_bundle_update: false)
```

Reply:
ugh, I missed one, thanks

Reply:
I need to test with a plugin

Reply:
Fixed.  I still need to run through plugin setup/update locally, with github actions and via travis.